### PR TITLE
 Change default_returns_func to use Quandl API

### PR DESCRIPTION
**Pull request by jaycode**
I updated `default_returns_func` to use Quandl, created `get_symbol_returns_from_quandl`, and deprecated `get_symbol_returns_from_yahoo`.

### CODE REVIEW

**Comment by twiecki** [comment]
Thanks @jaycode!

CC @richafrank for assigning a reviewer.

File: empyrical/utils.py [hunk 1]
```diff
@@ -458,15 +498,15 @@ def default_returns_func(symbol, start=None, end=None):
...
-                                  start='1/1/1970',
+                                  start=start,
                                   end=datetime.now())
-        rets = rets[start:end]
+        rets = rets[rets.index.isin(pd.bdate_range(start, end))]
     else:
-        rets = get_symbol_returns_from_yahoo(symbol, start=start, end=end)
-
+        rets = get_symbol_returns_from_quandl(symbol, start=start, end=end)
+    rets.sort_index(inplace=True)
```
Comment (twiecki) [inline empyrical/utils.py]: Does quandl not give a sorted index?

**Comment by twiecki** [comment]
@jaycode sorry for the long delay, but we're finally getting back to this.

**Comment by eigenfoo** [comment]
Hello @jaycode, this is a high-priority issue for us, so I'll be work on my own version of the PR, if that's alright with you. Thanks so much!

File: empyrical/utils.py (hunk 1 above)
Comment (eigenfoo) [reply empyrical/utils.py]: No. Curiously, it is returned in _reverse_ chronological order.

screen shot 2018-05-17 at 2 44 17 pm

File: empyrical/utils.py [hunk 2]
```diff
@@ -424,10 +431,43 @@ def get_symbol_returns_from_yahoo(symbol, start=None, end=None):
...
+    Returns
+    -------
+    pandas.DataFrame
+        Returns of symbol in requested period.
+    """
+
+    px = web.get_data_quandl(symbol, start=start, end=end)
+    rets = px[['AdjClose']]
+    rets = rets.shift(-1)
+    rets.iloc[-1]['AdjClose'] = px.tail(1)['AdjOpen']
```
[the remark this answers is not in the document]
Comment (eigenfoo) [reply empyrical/utils.py]: Done.

**Pull request closed without merging**